hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemFileStatus.java

```diff
@@ -21,6 +21,7 @@
 import java.io.IOException;
 
 import org.apache.hadoop.fs.CommonConfigurationKeys;
+import org.apache.hadoop.fs.FsStatus;
```

Comment:
can you move this down with the other org.apache.hadoop group? We try to keep the org.apache imports together, and while it's generally too late to move things around in committed code (backport pain amplification), it's good to try and follow the plan on newer imports

Reply:
Removed the import as the same is unused